### PR TITLE
IGNITE-14017 Willing to add s390x support to the docker image

### PR DESCRIPTION
**Pull request by vveider**

### CODE REVIEW

File: deliveries/docker/apache-ignite/s390x/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Base Image
+FROM  s390x/ubuntu:18.04
+
+ARG IGNITE_VER=2.9.1
```
Comment (vveider) [inline deliveries/docker/apache-ignite/s390x/Dockerfile:22]: Version hardcode is not recommended — it is difficult to maintain.
Instead either mandatory variable without default value or prepared Apache Ignite directory without version should be introduced.

File: deliveries/docker/apache-ignite/s390x/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+# The Author
+LABEL maintainer="LoZ Open Source Ecosystem (https://www.ibm.com/community/z/usergroups/opensource)"
+
+# Set Environment Variables
+ENV WORKDIR /root
```
Comment (vveider) [inline deliveries/docker/apache-ignite/s390x/Dockerfile:28]: `/opt` is better for custom sofware

File: deliveries/docker/apache-ignite/s390x/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+# Download AdoptOpenJDK 8
+ && cd $WORKDIR  \
+ && wget https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_s390x_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz \
+ && tar xvf OpenJDK11U-jdk_s390x_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz \
+# Install Apache Ignite
+ && cd $WORKDIR  \
+ && wget https://archive.apache.org/dist/ignite/${IGNITE_VER}/apache-ignite-${IGNITE_VER}-bin.zip \
+ && unzip -q apache-ignite-${IGNITE_VER}-bin.zip  \
+# Clean up cache data and remove dependencies that are not required
+ && apt-get remove -y \
```
Comment (vveider) [inline deliveries/docker/apache-ignite/s390x/Dockerfile:47]: apt clean should be done with apt install / update

File: deliveries/docker/apache-ignite/s390x/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+ENV PATH=$JAVA_HOME/bin:$WORKDIR/apache-ignite-${IGNITE_VER}-bin/bin:$PATH
+
+# Install Dependencies
+RUN apt-get update &&  apt-get install -y \
+        wget \
+        tar \
+        unzip \
+# Download AdoptOpenJDK 8
+ && cd $WORKDIR  \
+ && wget https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_s390x_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz \
```
Comment (vveider) [inline deliveries/docker/apache-ignite/s390x/Dockerfile:40]: Get, unpack and cleanup should be united in single layer.
And be separated with standalone RUN command

File: deliveries/docker/apache-ignite/s390x/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+ && apt-get autoremove -y \
+ && apt autoremove -y \
+ && apt-get clean && rm -rf /var/lib/apt/lists/*  $WORKDIR/apache-ignite-${IGNITE_VER}-bin.zip $WORKDIR/OpenJDK11U-jdk_s390x_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz
+
+# Copy sh files and set permission
+COPY run.sh $IGNITE_HOME/
+# Grant permission to copy optional libs
+RUN chmod 777 ${IGNITE_HOME}/libs
+# Grant permission to create work directory
+RUN chmod 777 ${IGNITE_HOME}
```
Comment (vveider) [inline deliveries/docker/apache-ignite/s390x/Dockerfile:59]: Can be united with chmod above

**Comment by imurchenko** [comment]
Looks good to me

**Pull request merged**